src/careamics/config/likelihood_model.py

```diff
@@ -10,8 +10,6 @@
     MultiChannelNoiseModel,
 )
 
-NoiseModel = Union[GaussianMixtureNoiseModel, MultiChannelNoiseModel]
```

Comment:
I'd keep this honestly unless you have a strong argument against it ;)

Reply:
No strong opinions, just wasn't working properly